Beey.DataExchangeModel.Common/Speakers/SpeakerField.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Beey.DataExchangeModel.Common.Speakers;
+
+/// <summary> Common well-known speaker field </summary>
+public readonly struct SpeakerField(string fieldName)
```

Comment:
Nerozumím záměru, protože tohle nic nezaručuje. Můžu si vyrobit jakékoliv jméno fieldu (dokonce i s null stringem, pokud vyrobím default hodnotu). Kdyby to byla třída s privátním konstruktorem, tak mám zajištěno, že budu moct používat pouze definované a validní hodnoty, ale takhle ne.

Reply:
Ano, jenže to je přesně účel :) Záměr je, aby se daly uložit a používat libovolné fieldy, nejen ty předdefinované. Ale napadlo mě, jak to upravit, aby to bylo ještě jasnější.

Reply:
Mrkni sem. Oddělil jsem [CommonSpeakerFields](https://github.com/newtontechnologies/beey-data-exchange-model/blob/speaker-catalog/Beey.DataExchangeModel.Common/Speakers/CommonSpeakerFields.cs), aby bylo zřejmé, že toto jsou jen ty běžné pole, ale jinak tam můžou být další pole.